vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -119,9 +119,13 @@ function Get-FilesToCacheOnVHD {
 
         foreach ($URL in $map[$dir]) {
             $fileName = [IO.Path]::GetFileName($URL)
-
+            # Doesnot cache containerd package in docker image
+            if ($containerRuntime -ne 'containerd' -And $dir -eq "c:\akse-cache\containerd\") {
+                Write-Log "Skip to download $URL for docker image"
+                continue
+            }
             # Windows containerD supports Windows containerD, starting from Kubernetes 1.20
-            if ($containerRuntime -eq 'containerd' -And $dir -eq "c:\akse-cache\win-k8s\") {
+            elseif ($containerRuntime -eq 'containerd' -And $dir -eq "c:\akse-cache\win-k8s\") {
```

Comment:
This if and the above if in L123 can be regarded as separate logic. Keep using if here makes the logics earlier

Reply:
if:
if:
Above means that both will be executed without considering any exit inside if.
if:
elseif:
Above means that only one will be executed in one condition.
Here both work but I think that it should be better to use if and elseif.

Reply:
My suggestion is `ifelse` involves more thinking because we need to consider the relationship between if and ifelse, in case they can be separated by two standalone clauses, use two if. Just a small suggestion. If you think the current status looks better, then keep it as it is.

Reply:
Make sense. Since we have use `if` in Test-FilesToCacheOnVHD, I updated the code from `elseif` to `if`.